build-tools/packages/build-cli/src/commands/generate/readme.ts

```diff
@@ -25,13 +26,22 @@ Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.
             ...(config.pjson.oclif.additionalVersionFlags ?? []).sort(),
         ];
         const versionFlagsString = `(${versionFlags.join("|")})`;
+        const version = semver.parse(config.version) ?? config.version;
```

Comment:
Is it actually acceptable to fail to parse here?  Would love if we can blow up instead.

Reply:
Done. Errors out now if the version is not parseable.